src/core/cgroup.c

```diff
@@ -34,6 +35,13 @@
  * out specific attributes from us. */
 #define LOG_LEVEL_CGROUP_WRITE(r) (IN_SET(abs(r), ENOENT, EROFS, EACCES, EPERM) ? LOG_DEBUG : LOG_WARNING)
 
+uint64_t tasks_max_scale(const TasksMax *tasks_max) {
```

Comment:
maybe call this `task_max_resolve()`? otherwise one might thing this returns `tasks_max->scale` in some mangled form, but actually it's more that it returns `tasks_mask->value`